pkgs/by-name/ca/caddy/package.nix

```diff
@@ -61,12 +66,83 @@ buildGoModule {
       --zsh <($out/bin/caddy completion zsh)
   '';
 
-  passthru.tests = {
-    inherit (nixosTests) caddy;
-    version = testers.testVersion {
-      command = "${caddy}/bin/caddy version";
-      package = caddy;
+  passthru = {
+    tests = {
+      inherit (nixosTests) caddy;
+      version = testers.testVersion {
+        command = "${caddy}/bin/caddy version";
+        package = caddy;
+      };
     };
+    withPlugins =
```

Comment:
Nit, but I think this is very long function to be defined at the body of the derivation. I suggest moving this to `with-plugins.nix` or something and importing the file here.

Reply:
I will not block PR for my nit, so if someone else wants to merge feel free.

Reply:
Done, moved to `plugins.nix` and use of `callPackage`. I think this is the right way, but maybe I should inherit more stuff?